tests/test_scan.py

```diff
@@ -0,0 +1,19 @@
+from unittest import TestCase
+
+from wifi.scan import Cell
+from wifi import subprocess_compat as subprocess
+from mock import MagicMock
+from test_parsing import output
```

Comment:
`output` is bad naming, also building it on a file and using in another does not help understanding it. So I'll suggest to move the join in this file, where needed. And call it maybe `expected_output`